.knip.json

```
@@ -0,0 +1,3 @@
+{
+  "ignoreDependencies": ["commitlint-config-bloq", "eslint-config-prettier"]
```

This can be removed

```suggestion
  "ignoreDependencies": ["eslint-config-prettier"]
```

See [the job's logs](https://github.com/hemilabs/better-sort-github-actions/actions/runs/14913042384/job/41891886244#step:6:10)

> Unused item in ignoreDependencies: commitlint-config-bloq

---

Nice catch!